main.cpp

```diff
@@ -1,10 +1,58 @@
 #include <iostream>
+#include <iomanip>   // format manipulation
 
 using namespace std;
-
+/**This program displays an equilateral triangle based on the user's input.
+ * The user decides on its size and the character that it may consist of.
+ * The triangle increases by three characters each time and is based on the 
+ * T(n)=2n+1 equation.*/
+void inputCheck(int & size)//Still trying to figure our cin.fail() and cin.good()
+{
+ bool isdigit();
+    if(size.isdigit()==true)
+        
+        cin.good();
+    cout<<endl;
+        
+    else
+        cin.fail();
+        cout << "Please re-enter a size. ";
+        cin.clear();
+        cin>>size;
+}
+void generateEquilateralTriangle(char character, int size)
+{
+     for (int i = 0; i < size; i++)
+    {
```

Comment:
Yes butttt put a space between ) and {. Should be `for (int j = 0; j < (2*size + 1); j++) {`. Also notice the space I added around `+ 1`. Make sure those maths are nicely spaced out. And also notice I did not add the space around the 2*size. Think of it in terms of math 2n + 1..